spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteTablePathsAction.java

```diff
@@ -940,16 +1024,20 @@ protected void checkFileNum(
             .load(result.fileListLocation())
             .as(Encoders.STRING())
             .collectAsList();
-    assertThat(filesToMove.stream().filter(f -> f.endsWith(".metadata.json")).count())
-        .withFailMessage("Wrong rebuilt version file count")
+    Predicate<String> isManifest = f -> (f.endsWith("-m0.avro") || f.endsWith("-m1.avro"));
```

Comment:
Nit: Is it the minimum parens?

can we do:

```
    Predicate<String> isManifest = f -> (f.endsWith("-m0.avro") || f.endsWith("-m1.avro"));
    Predicate<String> isManifestList = f -> (f.contains("snap-") && f.endsWith(".avro"));
    Predicate<String> isMetadataJSON = f -> f.endsWith(".metadata.json");
```

Sorry I dont have the local build setup yet, but just wondering as it still seems we can reduce a few

Reply:
yeah looks like I can remove more brackets and this works
```java
    Predicate<String> isManifest = f -> f.endsWith("-m0.avro") || f.endsWith("-m1.avro");
    Predicate<String> isManifestList = f -> f.contains("snap-") && f.endsWith(".avro");
    Predicate<String> isMetadataJSON = f -> f.endsWith(".metadata.json");
```